Jenkinsfile

```diff
@@ -29,7 +29,29 @@ node {
                 sh 'google-chrome --version'
                 sh 'yarn kbn bootstrap'
             }
+            
+            stage('Build test plugins') {
+                sh """
+                    echo " -> building kibana platform plugins"
+                    node scripts/build_kibana_platform_plugins \\
```

Comment:
This file path 'scripts/build_kibana_platform_plugins' does not exist in 7.4.2 nor does the kbn-optimizer package it requires in.

Reply:
The above script was removed and a new job started: https://jenkins.bfs.sichend.people.aws.dev/job/Kibana/job/bfs7.4.2/13/

Since the build test plugin is not in bfs7.4.2 branch, it has been removed